### PR TITLE
soap-hil: add scorch demo and options to run without hardware

### PR DESCRIPTION
**Pull request by cmulk**
New additions to soap-hil:
- Options to run without HIL (ironic)
- SCORCH pipeline demo including traffic capture, parsing via Zeek, and saving results to elastic

### CODE REVIEW

File: soap-hil/README.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
...
+This repository contains a topology to demonstrate the Hardware-In-the-Loop (HIL) capabilities of SCEPTRE. The topology is based on the original SOAP (SCEPTRE on a Platter) topology and underlying power model, simulated using PyPower. It includes the integration of a Siemens S7 PLC as HIL, functioning as a simple controller in the power model. _This topology also still fully functions without HIL and can be used by selecting a second, "virtual-only" control screen that is already included in the HMI configuration._
+
+In addition to the topology, this repo includes a demo attack against the PLC to manipulate and destablize the power system model. In addition, a SCORCH pipeline is included in the scenario which can be used to captur raw PCAP to exportable files, and packet metadata (via filebeat and [Zeek](zeek.org)) to an elastic instance.
```
Comment (GhostofGoes) [inline soap-hil/README.md:3]: `captur` -> `capture`

File: soap-hil/README.md [hunk 2]
```diff
@@ -99,65 +105,136 @@ Additionally, open a web browser (in the virtual machine), navigate to localhost
...
+After launching the experiment, enter the `attacker` machine and navigate to `/phenix/`.
+The script is runnable with
+```bash
+python3 plc-attack.py <STARTUP_DELAY_SECONDS> <NUMBER_REPEATS>
+```
+
+### Running an attack scenario in virtual-only mode
+A sample 'attack' is implemented through a few means to demonstrate SCEPTRE without a Hardware-In-The-Loop implementation.
+1. After launching the experiment, enter the `attacker` machine and launch `msfconsole`
+* Start by finding the applicable exploit `search ignition`
```
Comment (GhostofGoes) [inline soap-hil/README.md:186]: Indentation needed

File: soap-hil/README.md [hunk 3]
```diff
@@ -99,65 +105,136 @@ Additionally, open a web browser (in the virtual machine), navigate to localhost
...
+## Scorch run example
+1. When you are ready to begin an attack run, go to the Scorch tab for this experiment and click the red `Stopped` button to start a new run:
+    ![alt text](img/scorch-start.png)
+2. The Scorch pipeline should execute until you get to the `exp-break` breakpoint:
+    ![alt text](img/scorch-break.png)
+3. At this point you can execute one of the sample attacks described above.
+4. When finished, open the `exp-break` breakpoint and exit the terminal to allow the pipeline to continue:
+
+    ![alt text](img/scorch-break-exit.png)
+5. At this point the pipeline should complete and you can go back to `Files` tab of the main experiment page do view and download artifacts from the Scorch run:
```
Comment (GhostofGoes) [inline soap-hil/README.md:223]: `do view` -> `to view`

**Review by GhostofGoes — CHANGES_REQUESTED** [review]
LGTM. Just a few minor spelling and format issues I found while reading through it.

File: soap-hil/README.md (hunk 1 above)
Comment (cmulk) [reply soap-hil/README.md:3]: done

File: soap-hil/README.md (hunk 2 above)
Comment (cmulk) [reply soap-hil/README.md:186]: done

File: soap-hil/README.md (hunk 3 above)
Comment (cmulk) [reply soap-hil/README.md:223]: done

**Review by GhostofGoes — APPROVED** [review]

**Comment by GhostofGoes** [comment]
@cmulk LGTM, good to merge if you're ready.

**Pull request merged**